patches/react-native-svg-transformer+1.3.0.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/node_modules/react-native-svg-transformer/index.js b/node_modules/react-native-svg-transformer/index.js
+index ac1fe67..bbd077f 100644
+--- a/node_modules/react-native-svg-transformer/index.js
++++ b/node_modules/react-native-svg-transformer/index.js
+@@ -1,6 +1,11 @@
+ const { resolveConfig, transform } = require("@svgr/core");
+ const resolveConfigDir = require("path-dirname");
+ 
++// Implementation uses a fragile approach to figure out which babel transformer to use
++// Makes it uses the one for React Native CLI, which is what's used to bundle our app
++// https://github.com/kristerkari/react-native-svg-transformer/issues/333
++const babelTransformer = require("@react-native/metro-babel-transformer");
```

Comment:
Can you add this description to the readme.md as well

Reply:
addressed via https://github.com/digidem/CoMapeo-mobile/pull/195/commits/daf2f56083f551eb6654520048b465e821a5701e